guide/14-deep-learning/exploring_model_extensibility_for_named_entity_recognition.ipynb

```diff
@@ -0,0 +1,722 @@
+{
```

Comment:
<p><strong>the directory containing the code of&nbsp;</strong><code><strong>GLiNER</strong></code><strong>.</strong> </p><p> </p><p>What does the folder contain? </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/2112/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='35'/>

Reply:
<p>The folder contains the clone git repo of GLiNER. This will help to import the necessary classes. </p>